src/WPGraphQL.php

```diff
@@ -419,50 +421,112 @@ public function init_admin() {
 	 * @return void
 	 */
 	public static function show_in_graphql() {
```

Comment:
This whole file is getting pretty large. Might be time to break the post type / taxonomy registration stuff to a separate class.

If so what should we name it? (Extensions use `WPGraphQL/CoreSchemaFilters`, but that doesn't really mesh with our file structure.